src/Breadcrumb.js

```diff
@@ -7,25 +7,40 @@ const getPath = (crumbArr, index) => {
 }
 
 const BreadcrumbItem = ({ index, crumb, crumbArr, separator, activePath }) => {
-  const { label, onClick } = crumb
   const path = getPath(crumbArr, index)
   const isActive = path === activePath
-  const onBreadCrumbClick = () => onClick(path)
-  if (index === 0) {
-    return <div className='breadcrumb-item-container'>
-      <div className='breadcrumb-item-container-cell'>&nbsp;</div>
-      <div onClick={onBreadCrumbClick} className={`breadcrumb-item-container-cell-label${isActive ? '--active' : ''}`}>{label}</div>
-    </div>
+
+  const onBreadCrumbClick = (e) => {
+    e && e.preventDefault && e.preventDefault()
+    e && e.stopPropagation && e.stopPropagation()
+    typeof crumb.onClick === 'function' && crumb.onClick(path)
   }
-  return <div className='breadcrumb-item-container'>
-    <div className='breadcrumb-item-container-cell'>{separator}</div>
-    <div onClick={onBreadCrumbClick} className={`breadcrumb-item-container-cell-label${isActive ? '--active' : ''}`}>{label}</div>
-  </div>
 
+  const spacer = <div className='breadcrumb-spacer' />
```

Comment:
no corresponding CSS class added, and this could break the even alignment of the labels with the separators

Reply:
the spacer only would exist for the first item, and there shouldn't be any separator, nor non-breaking space character there. so an empty div holds the place. there is no CSS rule because there would be nothing to put, and empty rules are invalid.